flights/FlightApp.js

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
-import { createStackNavigator } from '@react-navigation/stack';
-import { NavigationContainer } from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+import {NavigationContainer} from '@react-navigation/native';
 
-import SignUp from './src/screens/SignUp'
+import SignUp from './src/screens/SignUp';
+import LogIn from './src/screens/LogIn';
+import MyFlights from './src/screens/MyFlights';
+import NewFlight from './src/screens/NewFlight';
 
-const Stack = createStackNavigator()
+const Stack = createStackNavigator();
 
 const FlightApp = () => {
   return (
-    <NavigationContainer >
-      <Stack.Navigator>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="SingUp">
```

Comment:
Shouldn't this be "SignUp"?

Reply:
Hahahahahahaha, right! Finger error. Thanks a lot! We'll modify it.

Reply:
Ready! We've already modified it! @Seal-Trainer